website/docs/r/network_peering.html.markdown

```diff
@@ -69,65 +75,99 @@ resource "aws_vpc_peering_connection_accepter" "peer" {
 ```hcl
 
 resource "mongodbatlas_network_container" "test" {
-  project_id       = local.project_id
+  project_id       = "${local.project_id}"
   atlas_cidr_block = "192.168.192.0/18"
   provider_name    = "GCP"
 }
 
-resource "mongodbatlas_private_ip_mode" "my_private_ip_mode" {
-  project_id = local.project_id
-  enabled    = true
-}
-
 resource "mongodbatlas_network_peering" "test" {
-  project_id     = local.project_id
-  container_id   = mongodbatlas_network_container.test.container_id
+  project_id       = "${local.project_id}"
+  atlas_cidr_block = "192.168.0.0/18"
+
+  container_id   = "${mongodbatlas_network_container.test.container_id}"
   provider_name  = "GCP"
-  network_name   = "myNetWorkPeering"
-  gcp_project_id = local.google_project_id
+  gcp_project_id = "${local.GCP_PROJECT_ID}"
+  network_name   = "default"
+}
 
-  depends_on = [mongodbatlas_private_ip_mode.my_private_ip_mode]
+data "google_compute_network" "default" {
+  name = "default"
 }
 
-resource "google_compute_network" "vpc_network" {
-  name = "vpcnetwork"
+resource "google_compute_network_peering" "peering" {
+  name         = "peering-gcp-terraform-test"
+  network      = "${data.google_compute_network.default.self_link}"
+  peer_network = "https://www.googleapis.com/compute/v1/projects/${mongodbatlas_network_peering.test.atlas_gcp_project_id}/global/networks/${mongodbatlas_network_peering.test.atlas_vpc_name}"
 }
 
-resource "google_compute_network_peering" "gcp_main_atlas_peering" {
-  name         = "atlas-gcp-main"
-  network      = google_compute_network.vpc_network.self_link
-  peer_network = "projects/${mongodbatlas_network_peering.test.atlas_gcp_project_id}/global/networks/${mongodbatlas_network_peering.test.atlas_vpc_name}"
+resource "mongodbatlas_cluster" "test" {
+  project_id   = "${local.project_id}"
+  name         = "terraform-manually-test"
+  num_shards   = 1
+  disk_size_gb = 5
+
+  replication_factor           = 3
+  auto_scaling_disk_gb_enabled = true
+  mongo_db_major_version       = "4.0"
```

Comment:
should we use 4.2?

Reply:
Yeah, should start moving all the examples to 4.2.  I'll start w/ here and update the rest soon.